### ISSUE TITLE
https://github.com/mermaid-js/mermaid/

### ISSUE THREAD

**Issue by r1gm** [webjars/webjars-classic#8]
hi, 

Please make mermaid.js available through webjar.

Best Regards

**Comment by jamesward** [comment]
This is available as an NPM WebJar:
```
"org.webjars.npm" % "mermaid" % "11.1.0"
```

Does that work for you?

**Comment by r1gm** [comment]
it's not working

No versions available for org.webjars.npm:khroma:jar:[2.1.0,3.0.0-0) within specified range
No versions available for org.webjars.npm:antfu__utils:jar:[0.7.10,0.8.0-0) within specified range

and to many dependency added to my project

**Comment by jamesward** [comment]
Yeah, it looks like it has a lot of dependencies. I've deployed `org.webjars.npm mermaid 11.2.1` which doesn't address the number of dependencies issue, but should work. So I'll see if I can create a Classic WebJar with no dependencies. I'll let you know how that goes.

**Comment by r1gm** [comment]
thank you James

**Comment by jamesward** [comment]
I looked into this and it appears that the only way they distribute built version of mermaid is via NPM. So I think that the NPM WebJar is the best option.  Did `11.2.1` work for you? It does have a bunch of transitive dependencies but they should all resolve.
